frontend/src/components/print/__tests__/repairPrintConfig.spec.js

```diff
@@ -396,11 +400,281 @@ describe('repairConfig', () => {
+      contents: [
+        {
+          type: 'Picasso',
+          options: {
+            periods: ['/periods/1a2b3c4d'],
+            orientation: 'L',
+            filter: defaultFilter,
+          },
+        },
+      ],
+      documentName: 'test camp',
+      options: {
+        pageNumbers: true,
+        pageSize: 'A4',
+      },
+      language: 'de-CH-scout',
+    }
+
+    // when
+    const result = repairConfig(config, ...args)
+
+    // then
+    expect(result).toEqual({
+      camp: '/camps/1a2b3c4d',
+      contents: [
+        {
+          type: 'Picasso',
+          options: {
+            periods: ['/periods/1a2b3c4d'],
+            orientation: 'L',
+            filter: defaultFilter,
+          },
+        },
+      ],
+      documentName: 'test camp',
+      options: {
+        pageNumbers: true,
+        pageSize: 'A4',
+      },
+      language: 'de-CH-scout',
+    })
+  })
+
+  test('allows disabling pageNumbers', async () => {
+    // given
+    const config = {
+      camp: '/camps/1a2b3c4d',
+      contents: [
+        {
+          type: 'Picasso',
+          options: {
+            periods: ['/periods/1a2b3c4d'],
+            orientation: 'L',
+            filter: defaultFilter,
+          },
+        },
+      ],
+      documentName: 'test camp',
+      options: {
+        pageNumbers: false,
+        pageSize: 'A4',
+      },
+      language: 'de-CH-scout',
+    }
+
+    // when
+    const result = repairConfig(config, ...args)
+
+    // then
+    expect(result).toEqual({
+      camp: '/camps/1a2b3c4d',
+      contents: [
+        {
+          type: 'Picasso',
+          options: {
+            periods: ['/periods/1a2b3c4d'],
+            orientation: 'L',
+            filter: defaultFilter,
+          },
+        },
+      ],
+      documentName: 'test camp',
+      options: {
+        pageNumbers: false,
+        pageSize: 'A4',
+      },
+      language: 'de-CH-scout',
+    })
+  })
+
+  test('adds missing pageSize option', async () => {
+    // given
+    const config = {
+      camp: '/camps/1a2b3c4d',
+      contents: [
+        {
+          type: 'Picasso',
+          options: {
+            periods: ['/periods/1a2b3c4d'],
+            orientation: 'L',
+            filter: defaultFilter,
+          },
+        },
+      ],
+      documentName: 'test camp',
+      options: {
+        pageNumbers: false,
+      },
+      language: 'de-CH-scout',
+    }
+
+    // when
+    const result = repairConfig(config, ...args)
+
+    // then
+    expect(result).toEqual({
+      camp: '/camps/1a2b3c4d',
+      contents: [
+        {
+          type: 'Picasso',
+          options: {
+            periods: ['/periods/1a2b3c4d'],
+            orientation: 'L',
+            filter: defaultFilter,
+          },
+        },
+      ],
+      documentName: 'test camp',
+      options: defaultOptions,
+      language: 'de-CH-scout',
+    })
+  })
+
+  const pageSizes = [('A5', 'A4', 'A3')]
+  pageSizes.forEach((pageSize) => {
```

Comment:
Cool, nitpick:
With vitest you have to get used to it, but the syntax for parametrized tests is:

```
describe('idToColor', () => {
  it.each([
    [[undefined, false], '#4d4d4d'],
    [[null, false], '#4d4d4d'],
    [['', false], '#4d4d4d'],
    [['0000000', false], '#900'],
    [['0000000', true], '#4d4d4d'],
    [['fffffff', false], '#992600'],
    [['Wrong input', false], '#900'],
  ])('maps %p to %p', (input, expected) => {
```

See https://github.com/ecamp/ecamp3/blob/9e921c611dd87e7a0d2aa081d572d0317bb85ef3/common/helpers/__tests__/colors.spec.js#L5 as on example.
Or search "it.each"

Reply:
Yeah I've been in the ruby testing world a lot... e4d3c1f